auth_test.py::SubmissionTest::test_voting_up
--------------------------------------------------------------------
Last Page: http://127.0.0.1:8000/
  Browser: firefox (104.0.2)
   Driver: geckodriver (0.31.0)
Timestamp: 1663916196  (Unix Timestamp)
     Date: Thursday, September 22, 2022
     Time: 11:56:36 PM  (Pacific Daylight Time, UTC-08:00)
--------------------------------------------------------------------
Traceback: File "c:\python38-32\lib\unittest\case.py", line 676, in run
    self._callTestMethod(testMethod)
  File "c:\python38-32\lib\unittest\case.py", line 633, in _callTestMethod
    method()
  File "C:\Users\Ian\Documents\Python Scripts\Bandnames\bandnames.cool\bnSubmission\tests\auth_test.py", line 89, in test_voting_up
    self.click("#delete-button0")
  File "c:\python38-32\lib\site-packages\seleniumbase\fixtures\base_case.py", line 283, in click
    element = page_actions.wait_for_element_visible(
  File "c:\python38-32\lib\site-packages\seleniumbase\fixtures\page_actions.py", line 421, in wait_for_element_visible
    timeout_exception(NoSuchElementException, message)
  File "c:\python38-32\lib\site-packages\seleniumbase\fixtures\page_actions.py", line 190, in timeout_exception
    raise exc(msg)
Exception: Message: 
 Element {#delete-button0} was not present after 7 seconds!
